services/test-data/pangaea/pangaea_geo_none.tsv

```diff
@@ -0,0 +1,118 @@
+/* DATA DESCRIPTION:
```

Comment:
For all inserted datasets: Did you check that we are allowed to use the data, e.g. CC0 or something similar?

Reply:
All files are: https://creativecommons.org/licenses/by/4.0/
I guess that is OK?